icicle v 1.0.0 docs (#10)
ingonyama-zk/developer-docs: docs/icicle/supporting-additional-curves.md

# Supporting Additional Curves

:::note

> Supporting additional curves is still a work in progress and is bound to change in the future. Currently, the process is very manual; however, we are going to automate the majority of it in future releases.

Reviewer: ```suggestion
Supporting additional curves is still a work in progress and is bound to change in the future. Currently, the process is very manual but plan to automate the majority of it in future releases.
```

:::

We understand the need for ZK developers to use different curves, some common some more exotic. For this reason we designed ICICLE to allow developers to add any curve they desire.

> Currently adding new curves is a bit manual but we will be improving the process in upcoming versions.

Reviewer: Remove, already noted above
